### ISSUE TITLE
Charts and graphs - Specific test cases: Keyboard support

### ISSUE THREAD

**Issue by martynka** [wet-boew/wet-boew#6853]
It looks to me that what looks like links to the alternative texts in the case of charts, tables, are not accessible with the keyboard only. Could anyone please comment on this? Charts and graphs - Simple and easy: http://wet-boew.github.io/wet-boew/demos/charts/charts-en.html.

**Comment by LaurentGoderre** [comment]
You are right! That is very weird....

**Comment by LaurentGoderre** [comment]
Seems like they are not correctly instantiated the way the "View Code" is

**Comment by LaurentGoderre** [comment]
@duboisp please fix asap

**Comment by martynka** [comment]
Thank you Laurent!

**Comment by martynka** [comment]
I forgot to include the following in my post: @wet-boew/access-working-group

**Comment by hsrudnicki** [comment]
@martynka @duboisp @LaurentGoderre and others: Are you ready to close this issue?

**Comment by LaurentGoderre** [comment]
No, this is still an issue
